src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -260,7 +274,9 @@ def _describe(self, catalog: str, database: str, table: str) -> Table | None:
                 upgraded_to=self._parse_table_props(describe.get("Table Properties", "").lower()).get(
                     "upgraded_to", None
                 ),
-                storage_properties=self._parse_table_props(describe.get("Storage Properties", "").lower()),  # type: ignore[arg-type]
+                storage_properties=self._parse_table_props(
+                    describe.get("Storage Properties", "").lower()
```

Comment:
The value can be None and this code will crash

Reply:
`get("Storage Properties", "")` would return empty string if the key does not exist, so we should be safe